### PR TITLE
Updating Linting Library from pedantic to flutter_lints

### PR DESCRIPTION
**Pull request by Sarbjyotsingh**

### CODE REVIEW

**Review by rutkayb — APPROVED** [review]

**Comment by rutkayb** [comment]
run tests

**Comment by hudson-pierce** [comment]
run tests

File: analysis_options.yaml [hunk 1]
```diff
@@ -1 +0,0 @@
-include: package:pedantic/analysis_options.yaml
```
Comment (MrDyro) [inline analysis_options.yaml:1]: You have to include the analysis_options from Flutter lints
Like this:
include: package:flutter_lints/flutter.yaml

**Review by MrDyro — CHANGES_REQUESTED** [review]

File: analysis_options.yaml (hunk 1 above)
Comment (Sarbjyotsingh) [reply analysis_options.yaml:1]: Resolved

**Review by MrDyro — APPROVED** [review]

**Pull request merged**